sodetlib/operations/power_amplifiers.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import sodetlib as sdl
+
+sdl.set_action()
+def fullbandresponse_metric(S, band):
+	"""
+	LNA check metric using S.full_band_resp
+
+	Args
+	----
+    S : SmurfControl
+        Pysmurf instance
+	bands : iterable
+		Band to use.
+
+    Returns
+    ----------
+    metric: flot
+    	Median value of the fft returned by full_band_resp
+    data: array
+		(f, resp) -- Raw data returned by full_band_resp
+	"""
+	f, resp = S.full_band_resp(band=b)
+	# Or do we need median of some frequency range???
+    return np.nanmedian(np.real(resp)), (f,resp) 
```

Comment:
White space issues, this needs to be fixed throughout. For docs formatting / white space I'd reference other sodetlib operations